src/parser.ts

```diff
@@ -124,7 +125,7 @@ export function getAllDataTypes(
 export function resolveApiModel(
   apiModel: model.document.BaseUnit,
```

Comment:
why is apiModel not a Document?

Reply:
Because it doesn't have to be. This function never uses `encodes` or `declares`, so we don't want to use a type that requires those properties.